client/src/__tests__/modal.test.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { configure } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import { shallow } from 'enzyme';
+import Modal from '../components/Modal.jsx';
+
+configure({ adapter: new Adapter() });
+
+it('should render correctly when props.showModal is true', () => {
+  const props = {
+    showModal: true
+  };
+  const modal = shallow(<Modal showModal={props.showModal}/>);
+  
+  expect(modal).toMatchSnapshot();
```

Comment:
How much coverage do these Snapshot tests do?

Reply:
my test coverage is surprisingly high but obviously these tests aren't the most valuable. i need to add more